docs/workflow/testing/libraries/filtering-tests.md

```diff
@@ -176,3 +234,67 @@ dotnet build <csproj_file> /t:Test /p:TargetOS=windows
 ```sh
 dotnet build <csproj_file> /t:Test /p:TargetOS=OSX /p:WithCategories="OuterLoop;failing""
 ```
+
+## SkipOnCoreClrAttribute
```

Comment:
We also `SkipOnMonoAttribute`. 

On average, there are more tests disabled on Mono than on CoreCLR, so `SkipOnMonoAttribute` is more common.